Example_Programs/calculator_complex/Makefile.txt

```diff
@@ -0,0 +1,57 @@
+PROJ_NAME = Complex_Calculator
```

Comment:
No need of this file, when you already have Makefile

Reply:
I was confused with few commands written on that file and is looking forward to getting it cleared asap.

Reply:
Done!